### PR TITLE
Landing UI arup button - XYZ fix

### PR DESCRIPTION
**Pull request by ninehundred**
fixed by changing variable names - seems there was a caching issues with the servers object.

### CODE REVIEW

File: src/store/speckle/speckleUtil.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ const REFRESH_TOKEN = `${APP_NAME}.RefreshToken`;
 const SERVER = `${APP_NAME}.Server`;
 
 export function goToSpeckleAuthpage(server: Server) {
+  console.log(`server arg coming into goToSpeckleAuthpage\n`, server)
```
Comment (Exilliar) [inline src/store/speckle/speckleUtil.ts:31]: console.log to remove

File: src/store/speckle/speckleUtil.ts [hunk 2]
```diff
@@ -47,7 +48,7 @@ export function speckleLogOut() {
 
 // Exchanges the provided access code with a token/refreshToken pair, and saves them to local storage.
 export async function exchangeAccessCode(accessCode: string, server: Server) {
-  const res = await fetch(`${server.url}/auth/token/`, {
+  const res = await fetch(`${server.url}/steve/auth/token/`, {
```
Comment (Exilliar) [inline src/store/speckle/speckleUtil.ts:51]: ? Was this used for testing?

**Review by Exilliar — COMMENTED** [review]
I think that there might be some problems with speckle atm preventing the actual log in from working. Hopefully that will start working soon. Have a couple of comments from the code though, nothing major

File: src/store/speckle/speckleUtil.ts (hunk 1 above)
Comment (ninehundred) [reply src/store/speckle/speckleUtil.ts:31]: fixed

**Comment by ninehundred** [comment]
added vue-template-es2015-compiler. May require a bit more of a look at later though as I didn't need this before today on mac or on other serves. Appears to all be working now with the new module installed though.

**Review by Exilliar — COMMENTED** [review]
I was still getting build errors. It was giving me errors to do with Vite, which is Vue 3's build tool thing. The way that I fixed it was to do
```
## remove node_modules
git checkout origin/dev -- package-lock.json package.json
npm i
```
That should hopefully mean that you don't need to add that es2015 compiler line to the package.json.

There's also one console.log to remove

**Review by Exilliar — APPROVED** [review]
all working. Good job!

**Comment by Exilliar** [comment]
looks like the branch just needs updating, should just mean doing a `git pull origin dev`. Doesn't look like there are any merge conflicts so it should work fine

**Pull request merged**